### PR TITLE
Update and refresh Publish packages

### PR DESCRIPTION
**Pull request by v-thepet**
User Story https://dev.azure.com/mseng/TechnicalContent/_workitems/edit/1973816

### CODE REVIEW

**Comment by v-thepet** [comment]
@Mikejo5000 @nkolev92 @zivkan @JonDouglas - This article is ready for your review. The article has a couple of includes that were edited in PR.https://github.com/NuGet/docs.microsoft.com-nuget/pull/2862. Please review the includes in that PR. Once that PR is merged, I'll uncomment the includes calls in this article. Thanks for your timely review, but please don't merge this PR until the other PR is merged and the includes calls in this one are uncommented.

Note the warnings caused by invalid author/ms.author in two files that aren't changed in this PR. I started getting these warnings in all my open PRs as of last night 8/29. The author/ms.author are invalid in the upstream, but several other files in the repo also have these invalid names, and aren't being pulled in to PRs. I can't explain it. Please review this article and I'll work on getting the extra files out of the PRs. Thanks.

**Comment by v-thepet** [comment]
@Mikejo5000 @nkolev92 @zivkan @JonDouglas - I changed my mind and uncommented the includes. I didn't want to risk this getting merged with them commented out. When reviewing the sections **Create an API key** and **Use the dotnet CLI**, please make any edits in the edited includes at PR #2862. Thanks.

**Review by zivkan — APPROVED** [review]

**Comment by zivkan** [comment]
Since this docs page is in the `nuget-org` directory, I'll wait a day to see if anyone from the server team wants to review.

**Review by joelverhagen — APPROVED** [review]

File: docs/nuget-org/Publish-a-package.md
Comment (zhhyu) [inline docs/nuget-org/Publish-a-package.md:21]: media/publish_NuGetSignIn.png -> media/publish_NuGetSignIn.PNG?

Seems that the image path is broken.

**Review by zhhyu — APPROVED** [review]

File: docs/nuget-org/Publish-a-package.md
Comment (v-thepet) [reply docs/nuget-org/Publish-a-package.md]: @zhhyu - It works for me, but the uppercase extension is nonstandard and has caused problems in other repos so I changed it (for both graphics). (Usually the only way to do this from Windows machines is to rename the graphics.)

**Comment by v-thepet** [comment]
@zivkan @nkolev92 - This should be ready to merge, thanks.

**Pull request merged**
